### PR TITLE
Fix/banxa and bridges

### LINKED ISSUE #770: Gateway : banxa should be available only for mainnet
## Issue Type

```
[ x ] bug report
[ ] feature request
```

## Current Behavior

1. Banxa should only be enabled for ETHEREUM MAINNET only.
2. Remove the third-party bridge from TestNet as it's not supported.

## Steps to Reproduce

1.
2.
3.

### PR DESCRIPTION
**Pull request by sk-enya**
:clipboard: Add associated issues, tickets, docs URL here.

Closes: https://github.com/bobanetwork/boba/issues/770

## Overview

Disable the bridges for Tesetnet

## Changes

Describe your changes and implementation choices. More details make PRs easier to review.

- Add check for current network type on AvailableBridges component
- Migrated AvailableBridge component to typescript with styles-component in place for styling.
- Prepare util function to fetch bridges based on the token instead of fetching from network service (clean up)
- Create types for available bridges.
- Added checks on for isBanxaEnable, Testnet, Mainnet, Available Bridge
- Disable Banxa for Testnet as it's taking time to set up sandbox, which is good but not mandatory.
- Add unit test cases for Available bridges component.
- Left / Right Scrollbar issue fixed.

## Testing

Describe how to test your new feature/bug fix and if possible, a step-by-step guide on how to demo this.
- On connecting to the gateway mainnet you will see the Banxa Link on the bridges page.
- If connected to any other network except Ethereum on Mainnet or Testnet `Banxa` won't be enabled.

### CODE REVIEW

**Review by alvaro-ricotta — APPROVED** [review]

**Comment by InoMurko** [comment]
Why am I not seeing any tests? :-D

**Comment by sk-enya** [comment]
> Why am I not seeing any tests? :-D

Hi, @InoMurko I have just updated the PR by migrating the component to ts and added unit test cases.

File: packages/boba/gateway/src/components/availableBridges/tests/index.test.tsx [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    >
+      <Theme>
+        <AvailableBridges {...{ token, walletAddress }} />
+      </Theme>
+    </Provider>
+  )
+}
+
+describe('Avaialble Bridges', () => {
+  let token = {
```
Comment (InoMurko) [inline packages/boba/gateway/src/components/availableBridges/tests/index.test.tsx]:
```suggestion
describe('Available Bridges', () => {
  let token = {
```

**Review by InoMurko — APPROVED** [review]
EXCELLENT

**Review by alvaro-ricotta — APPROVED** [review]

**Pull request merged**